src/App.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
 
 const App = () => (
-  <h1>React sum</h1>
+  <body>
```

Comment:
don't use body in any React component, your markup renders into a div inside an existing body
![image](https://user-images.githubusercontent.com/57598825/104306394-8d94b600-54d6-11eb-8f1a-cc5d72106025.png)

Reply:
and what about this one?

Reply:
Sorry 😬 Forgot to add the App.js file to the new commit